Gemfile.lock

```diff
@@ -1,13 +1,3 @@
-GIT
-  remote: https://github.com/sgruhier/foundation_rails_helper.git
-  revision: 45709e28f567b2e619a06091eb63d4f050f861d8
-  specs:
-    foundation_rails_helper (3.0.0)
```

Comment:
Can we leave it ass 3.0 until https://github.com/sgruhier/foundation_rails_helper/issues/181 is fixed 🙏🏽 ? It shouldn't be any difference, but I prefer it that way. Thanks

Reply:
@andreslucena , foundation_rails_helper v3 has a strong dependency on rails < 6 as we can see here : https://rubygems.org/gems/foundation_rails_helper/versions/3.0.0 

I have manually made a diff between the github master branch and the downloadable v 4 gem, and I confirm that the only difference is the Version parameter as commented on: https://github.com/sgruhier/foundation_rails_helper/issues/181#issuecomment-1003771444 

This is the only gem i can think of that is actually being installed from github. My recommendation is to upgrade and use the ruby gems version.

Reply:
Ok, I don't like it, but I too checked it and as far as I see this would be the easy way of solving this warning. Thanks for the tip of using Meld, I didn't know it 😄